server.js

```diff
@@ -95,10 +95,26 @@ app.post('/api/v1/vineyards', (request, response) => {
 });
 
 app.put('/api/v1/vineyards/:id', (request, response) => {
-  // fix information assuming correct entry-- Block efforts to change the id
-  //return (201) for successful update return id
-  // return (?ERROR) if cant successfully update, 
-  //catch 500
+  const updatedVineyard = request.body;
+  let id  = parseInt(request.params.id)
+
+database('vineyards').select()
+  .then(vineyards => {
+    let foundVineyard = vineyards.find(vineyard => {
+      return vineyard.id === id
+    })
+    if(foundVineyard) {
+      database('vineyards').where('id', id).update(updatedVineyard)
+      .then(vineyard => {
+        response.status(201).json({id: id})
```

Comment:
Do you think we should make this response a bit more explicite? Possibly something along the lines of "Vineyard with the id of ${id} was successfully updated."